frontend/src/lib/components/NavBarComponents/navBarButton.tsx

```diff
@@ -14,44 +16,47 @@ export type NavBarButtonProps = {
      * An alternate icon to use when the button is in it's "active" state
      */
     activeIcon?: React.ReactNode;
-    /**
-     * Collapses the button (rendering only the icon)
-     */
-    collapsed?: boolean;
     /**
      * Renders the button in it's active state
      */
     active?: boolean;
-    /**
-     * Text shown next to the button
-     */
-    text?: string;
     /**
      * Tooltip text
      */
     title?: string;
+    /**
+     * Tooltip text when disabled
+     */
+    disabledTitle?: string;
 };
 
 function NavBarButtonComponent(
     props: NavBarButtonProps & ButtonProps,
     ref: React.ForwardedRef<HTMLDivElement>,
 ): React.ReactNode {
-    const { icon, activeIcon, text, collapsed, active, ...baseProps } = props;
+    const { icon, activeIcon, active, disabledTitle, ...baseProps } = props;
```

Comment:
Add `title` to destructured props, as usage of `baseProps` below adds both `Tooltip`-text and regular `title`-functionality for the `Button` when using `{...baseProps}`